src/CommunicationLayer/CommunicationContainer.cpp

```diff
@@ -3,7 +3,7 @@
 #include "../DataLayer/DataContainer.h"
 #include "CommunicationContainer.h"
 #include "CommDeviceControl/ConnectionController.h"
-#include "CommDeviceControl/CommDeviceManager.h"
```

Comment:
The reason this existed before was so that we could switch between serial and internet inputs. Instead of replacing the commDeviceManager and removing this capability, it would've been better to have the CommDeviceManager adapt to use the new internetCommDevice.